### PR TITLE
Book list

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary
- Added a new view in HTML to the book list
- Added some styles to format the grid of books
- Added new styles for titles and subheads

## Screenshot
Captura de Pantalla 2021-05-11 a la(s) 7 01 10 p  m

## Trello Card
https://trello.com/c/5JrafDPh/30-listado-de-libros

### CODE REVIEW

File: books/scss/list.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      width: 142px;
+    }
+
+    .book-info {
+      display: flex;
+      flex-direction: column;
+      margin-top: 10px;
+    }
+  }
+}
```
Comment (mcallegari10) [inline books/scss/list.scss]: Remember to add the EOF line

File: books/scss/list.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.book-list {
+  align-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 888px;
```
Comment (mcallegari10) [inline books/scss/list.scss]: Are Yyu sure about that size?

File: books/scss/list.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  width: 100%;
+
+  .book {
+    @include shadow(4px 4px 15px 2px $black-shadow);
+    align-self: flex-start;
+    flex: 0 1 20%;
```
Comment (mcallegari10) [inline books/scss/list.scss]: Why this flex?

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: books/scss/list.scss (hunk 2 above)
Comment (Julianhm9612) [reply books/scss/list.scss]: Have been improved

File: books/scss/list.scss (hunk 1 above)
Comment (Julianhm9612) [reply books/scss/list.scss]: Have been improved

File: books/scss/list.scss (hunk 3 above)
Comment (Julianhm9612) [reply books/scss/list.scss]: Was changed to a max-width 😄

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**